src/System.Xml.XDocument/tests/xNodeBuilder/CommonTests.cs

```diff
@@ -3400,7 +3401,9 @@ public void CData_4()
                 }
 
                 //[Variation(Id = 5, Desc = "WriteCData with ]]>", Priority = 1)]
-                public void CData_5()
+                [Fact]
+                [ActiveIssue(4054, default(PlatformID))]
```

Comment:
The goal was to annotate the test with ActiveIssue without disabling it. These are tests which are not failing, but are testing design decisions which need to be reevaluated.